packages/neo-one-website/docs/0-installation/1-environment-setup.md

```diff
@@ -58,6 +58,13 @@ In addition to the above, if you're developing TypeScript smart contracts using
  - `@neo-one/smart-contract-test` - TypeScript smart contract testing utilitiees.
  - `@neo-one/smart-contract-typescript-plugin` - TypeScript language server plugin for inline compiler diagnostics in your favorite IDE.
 
+### Updating
+
+```bash
+#npm update @neo-one/server
+yarn upgrade @neo-one/server
```

Comment:
This should be `yarn upgrade @neo-one/cli`

Reply:
Now that I think about it - everywhere in this PR where we use `@neo-one/server` should actually be `@neo-one/cli`. Users will likely never install `@neo-one/server` directly, it's a transitive dependency of `@neo-one/cli` which they will install. E.g. the spinner which says that it's checking for `@neo-one/server` updates will just confuse users since they never installed `@neo-one/server` themselves.

Reply:
👍